<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editTestStrainsDialog</class>
 <widget class="QDialog" name="editTestStrainsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StreptoDB - TestStrains</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bttn_addnew">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>add new Test-Strain</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Test-Strains:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_test">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>testing...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>301</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bttn_delete">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>delete selected</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_strainID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_strainName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_strainBrothID">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>if you delete a selected 
Test-Strain, all data for 
that Test-Strain will be 
deleted.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>editTestStrainsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>editTestStrainsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttn_addnew</sender>
   <signal>clicked()</signal>
   <receiver>editTestStrainsDialog</receiver>
   <slot>addItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bttn_delete</sender>
   <signal>clicked()</signal>
   <receiver>editTestStrainsDialog</receiver>
   <slot>deleteItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>insertIntoDB()</slot>
  <slot>addItem()</slot>
  <slot>deleteItem()</slot>
 </slots>
</ui>
